src/seedu/addressbook/AddressBook.java

```diff
@@ -383,12 +418,175 @@ private static String executeCommand(String userInputString) {
             return getUsageInfoForAllCommands();
         case COMMAND_EXIT_WORD:
             executeExitProgramRequest();
+        case COMMAND_SORT_WORD:
+            return sortAddressBook();
+        case COMMAND_EDIT_WORD:
+            return executeEditPerson(commandArgs);
         default:
             return getMessageForInvalidCommandInput(commandType, getUsageInfoForAllCommands());
         }
     }
 
     /**
+     * Executes the command to edit a person, by first checking if the person is valid, then editting the values of the person
+     *
+     * @param commandArgs  raw input from user
+     * @return message for successful editting of a person
+     */
+    private static String executeEditPerson(String commandArgs) {
+        Optional<String[]> person = checkValidPersonToEdit(commandArgs);
+        if (!person.isPresent()){
+            return "returning to edit screen";
+        }
+        final String[] personToAdd = rebuildPerson(person.get());
+        return getMessageForSuccessfulEditPerson(personToAdd);
+        
+	}
+    
+    /**
+     * Shows details about the person successfully editted 
+     *
+     * @param addedPerson - String[] representation of a person
+     * @return formatted string to declare a person has been successfully added
+     */
+	private static String getMessageForSuccessfulEditPerson(String[] addedPerson) {
+        return String.format(MESSAGE_EDITED,
+                getNameFromPerson(addedPerson), getPhoneFromPerson(addedPerson), getEmailFromPerson(addedPerson), getSchoolFromPerson(addedPerson));
```

Comment:
this is a particularly long line, consider splitting it to multiple lines in accordance with the coding standards.